src/dialogue choices.js

```diff
@@ -0,0 +1,226 @@
+/**
+❔
+@file dialogue choices
+@summary Present options to the user via the "say" interface, then react to their choice.
+@license MIT
+@author Violgamba (Jon Heard)
+
+
+@description
+Present options to the user via the "say" interface, then react to their choice.  This can be useful
+in a variety of ways, such as for creating dialogue trees or for picking settings.
+
+
+HOW TO USE - BASIC CHOICE RESULTS:
```

Comment:
although this says "basic" results, this includes some fairly involved setup for triggering locations. i'd recommend explaining that separately below, and keeping this example minimal